src/main/java/frc/robot/subsystems/Drivebase.java

```diff
@@ -181,11 +181,23 @@ public void setSlot(int slot) {
         right.selectProfileSlot(slot, 0);
     }
 
+    /**
+     * @deprecated use driveTank
```

Comment:
Why was this deprecated?

Reply:
There are now prewritten functions in the abstract class which do the same